deepspeech/main.go

```diff
@@ -27,6 +27,15 @@ var audio    = flag.String("audio", "",    "Path to the audio file to run (WAV f
 var lm       = flag.String("lm", "",       "Path to the language model binary file")
 var trie     = flag.String("trie", "",     "Path to the language model trie file created with native_client/generate_trie")
 var version  = flag.Bool("version", false, "Print version and exits")
+var extended = flag.Bool("extended", false, "Use extended metadata")
+
+func metadataToString(m *astideepspeech.Metadata) string {
```

Comment:
In GO, you can add a `String() string` method to `*astideepspeech.Metadata` (that implements the `Stringer` interface). That way, when you feed it to a `print` implementation as a `%s`, it will use this method.

Therefore you should move this logic in such a method in the `astideepspeech` package.

Reply:
Except that we prefer this code to live on the client side, this way people see it directly being manipulated, like we did in all other bindings.

Reply:
OK, makes sense